DiscordAlertsTrader/read_hist_msg.py

```diff
@@ -0,0 +1,519 @@
+import pandas as pd
+import re
+import json
+from datetime import datetime, timezone, timedelta
+from DiscordAlertsTrader.message_parser import parse_trade_alert
+from DiscordAlertsTrader.server_alert_formatting import (
+    format_alert_date_price,
+)
+
+def format_0dte_weeklies(contract, message_date, remove_price=True):
+    "remove price when stc title is bto"
+    if "0DTE" in contract.upper():
+        msg_date = message_date.strftime('%m/%d')
+        contract = re.sub(r"0DTE", msg_date,contract, flags=re.IGNORECASE)
+        if remove_price:
+            contract = contract.split(" @")[0]
+    elif "weeklies" in contract.lower():
+        msg_date= message_date
+        days_until_friday = (4 - msg_date.weekday() + 7) % 7
+        msg_date += timedelta(days=days_until_friday)
+        msg_date = msg_date.strftime('%m/%d')
+        contract =  re.sub(r"Weeklies", msg_date,contract, flags=re.IGNORECASE)
+        if remove_price:
+            contract = contract.split(" @")[0]
+    return contract
+
+def kent_formatting(message):
+    """
+    Reformat Discord message from Kent
+    """
+    alert = ''
+    for mb in message['embeds']:
+        if mb['description']:
+            alert += mb['description']
+    return alert
+
+import webcolors
+from discordalerts.lib.util import get_stock_signal_from_rgb
```

Comment:
where is that coming from?

Reply:
this is a locally installed python lib that i created

Reply:
this can not be in the package if the user can't have access to it.

Reply:
added instructions to install this library